modules/core/src/test/java/org/apache/ignite/internal/processors/cache/store/GridCacheWriteBehindStoreSelfTest.java

```diff
@@ -106,6 +106,42 @@ public void testSimpleStoreWithoutCoalescing() throws Exception {
         testSimpleStore(false);
     }
 
+    /**
+     * Checks that write behind cache flush frequency was correctly adjusted to nanos
+     * Expecting putAllCnt to be less or equal than elapsed time divided by flush frequency
+     *
+     * @throws Exception If failed.
+     */
+    public void testSimpleStoreFlushFrequencyWithoutCoalescing() throws Exception {
+        initStore(1, false);
+
+        long writeBehindFlushFrequencyNanos = FLUSH_FREQUENCY * 1000 * 1000;
+        int threshold = store.getWriteBehindStoreBatchSize() / 10;
+
+        try {
+            long start = System.nanoTime();
+
+            for (int i =0; i < threshold / 2; i++)
```

Comment:
seems code reformat (including automatic) will be useful here

Reply:
Fixed.